shell/common/test_renderer_context.h

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "renderer_context_manager.h"
+
+namespace flutter {
+namespace testing {
+
+//------------------------------------------------------------------------------
+/// The renderer context used for testing
+class TestRendererContext : public RendererContext {
+ public:
+  //------------------------------------------------------------------------------
+  /// Represents the current alive context.
+  ///
+  /// -1 represents no context has been set.
+  static int currentContext;
```

Comment:
This reads like an ivar but is a static. Also, see the naming convention rules https://google.github.io/styleguide/cppguide.html#Variable_Names.

Also, you don't need to use a static variable for this. We use GTEST fixtures for this purpose. Here is how: You can make `TestRendererContext` inherit from `::testing::Test`. This will allow you to use a TEST_F to write a test. Now each test gets its own context and is now isolated from sharing state with other tests and is more self contained.